src/game/client/system/particlesystem/particlesys.cpp

```diff
@@ -65,6 +64,12 @@ ParticleSystem::ParticleSystem(const ParticleSystemTemplate *temp, ParticleSyste
     m_saveable(true),
     m_unkBool1(false)
 {
+    m_lastPos.Zero();
+    m_pos.Zero();
+    m_velCoefficient.Zero();
```

Comment:
This Zero() appears unnecessary if x,y,z are changed to 1 one line later.

Reply:
fixed.